spec/controllers/line_items_controller_spec.rb

```diff
@@ -32,9 +32,13 @@
       order = create(:completed_order_with_totals)
       item = create(:line_item, order: order)
       while !order.completed? do break unless order.next! end
+      order.reload
```

Comment:
it's a bit hard to understand why we need the `while do` loop and the `order.reload`. Is this an issue at factory level?

Should we add some doc somewhere to explain why both are needed? Thanks for your help!